### PR TITLE
initialized docker files

### PR DESCRIPTION
**Pull request by moficodes**
started working on the kubernetes / knative pattern.

### CODE REVIEW

File: src/destination/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:10.16.1-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY ./ ./
```
Comment (jjasghar) [inline src/destination/Dockerfile]: best practice is to be more explicit here.

**Review by jjasghar — CHANGES_REQUESTED** [review]

**Comment by jjasghar** [comment]
Also we should add a build step for each container in the travis.yml, if anything to make sure it still builds with any changes up or down the stack.

File: src/destination/Dockerfile (hunk 1 above)
Comment (moficodes) [reply src/destination/Dockerfile]: I added a .dockerignore 🤔

Comment (moficodes) [reply src/destination/Dockerfile]: https://stackoverflow.com/a/54616645/10272405

As of now I dont think there is any way to add multiple folder with out using multiple COPY command, in which case we will add bunch of layers. I think the dockerignore makes the most sense.

**Review by moficodes — COMMENTED** [review]
Added 3 different COPY statement to the dockerfile to explicitly describe what is being copied.

**Review by jjasghar — APPROVED** [review]

**Pull request merged**